lib/driver.js

```diff
@@ -322,6 +322,18 @@ class AndroidDriver extends BaseDriver {
     Object.assign(this.opts, launchInfo);
     Object.assign(this.caps, launchInfo);
 
+    // Uninstall any uninstallOtherPackages which were specified in caps
+    if (this.opts.uninstallOtherPackages) {
+      let uninstallOtherPackages;
+      try {
+        uninstallOtherPackages = helpers.parseArray(this.opts.uninstallOtherPackages);
+      } catch (e) {
+        log.errorAndThrow(`Could not parse "uninstallOtherPackages" capability: ${e.message}`);
+      }
+      // Only SETTINGS_HELPER_PKG_ID package is used by UIA1
```

Comment:
how about the bootstrap jar?

Reply:
I thought so, but I was able to get only the setting package